public/views/settings.html

```diff
@@ -0,0 +1 @@
+Some settings go here
```

Comment:
Is settings like admin setting?  Just trying to clarify what the purpose of this page will be.